### PR TITLE
Incorporated changes for OpenJ9 due to the update of OpenJDK RHEL8 images

### PR DESCRIPTION
**Pull request by mdrafiur**
…ages

Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request title is properly formatted: `[CLOUD-XYA] Subject`
- [ ] Pull Request contains link to the JIRA issue
- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: openj9-11-rhel8.yaml [hunk 1]
```diff
@@ -51,10 +51,10 @@ modules:
     version: "7"
   - name: jboss.container.jolokia
     version: "7"
-  - name: jboss.container.java.s2i.bash
+  - name: jboss.container.dnf
   - name: jboss.container.maven
-    version: "8.0.3.5"
-  - name: jboss.container.java.singleton-jdk
```
Comment (jmtd) [inline openj9-11-rhel8.yaml:57]: I haven't tried a build yet to test, but you might still want this module (at the end of the list) as I think maven will still pull in openjdk.

**Comment by jmtd** [comment]
Please target the `develop` branch initially. I can change the PR target here, but it will pull in a lot of other commits, so better if you rebase locally and force push first. Thanks!

**Comment by mdrafiur** [comment]
@jmtd Could you please confirm about rebasing the `develop` branch as the image was built using the `release` branch. Thanks

**Comment by jerboaa** [comment]
@mdrafiur I'm not Jon, but we first create PRs for the `develop` branch and then merge it down to `release` once integrated into `develop`.

**Comment by jerboaa** [comment]
Looks like https://github.com/jboss-container-images/openjdk/pull/141/ does this. I'd suggest to close this one and ask for a merge down to `release` once #141 has been integrated.

**Comment by mdrafiur** [comment]
Thanks @jerboaa . Closing the PR and as you suggested, will ask for a merge down to release once #141 has been integrated.

**Comment by jmtd** [comment]
Hi @mdrafiur, @jerboaa is right about our workflow, but I realise this might not be a good fit for OpenJ9. I'll raise a JIRA so we can discuss the workflow separately.

**Pull request closed without merging**